networking/bgp_routing/about-bgp-routing.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-bgp-routing"]
+= About BGP routing
+include::_attributes/common-attributes.adoc[]
+:context: about-bgp-routing
+
+toc::[]
+
+This feature provides native BGP routing capabilities for the cluster. Additionally, this feature must be enabled to use the RouteAdvertisements feature of the OVN-Kubernetes network plugin.
+
+include::modules/nw-bgp-about.adoc[leveloffset=+1]
+include::modules/nw-metallb-frr-k8s-configuration-crd.adoc[leveloffset=+1]
+//include::modules/nw-bgp-routeadvertisements-object.adoc[leveloffset=+1]
+//include::modules/nw-bgp-examples.adoc[leveloffset=+1]
```

Comment:
I guess these two are commented out. `nw-bgp-routeadvertisements-object.adoc` has an `omit` but not `nw-bgp-examples.adoc`. Those are really routeadvertisements examples so they should be omitted as well.